oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/measure/BanyanDBTopologyQueryDAO.java

```diff
@@ -129,8 +129,6 @@ List<Call.CallDetail> queryServiceRelation(Duration duration,
             MeasureQueryResponse resp = query(modelName,
                                               ImmutableSet.of(
                                                   ServiceRelationClientSideMetrics.COMPONENT_IDS,
-                                                  ServiceRelationClientSideMetrics.SOURCE_SERVICE_ID,
-                                                  ServiceRelationClientSideMetrics.DEST_SERVICE_ID,
```

Comment:
@lujiajing1126 @hanahmily Please review this class.

These columns are actually not used in final analysis. Entity ID should be enough for decoding.

BTW, BanyanDB implementation is not using `Aggregation` query, so with 30-60 min, we face a huge dataset of query result. Please recheck.